peer/brontide.go

```diff
@@ -947,12 +947,25 @@ func (p *Brontide) loadActiveChannels(chans []*channeldb.OpenChannel) (
 		// routing policy into a forwarding policy.
 		var forwardingPolicy *models.ForwardingPolicy
 		if selfPolicy != nil {
+			var inboundWireFee lnwire.Fee
+			_, err := selfPolicy.ExtraOpaqueData.ExtractRecords(
```

Comment:
I don't think it is necessary, because we aren't parsing wire data here. If we would not trust our own data, there are probably an endless number of other places where we parse tlv in an 'unsafe' way?

Reply:
Good point. I've removed safe extract from `FeeReport`, it shouldn't be necessary indeed. And don't want to sweep any potential unexpected errors under the rug.